src/utils/zcl_abapgit_utils.clas.abap

```diff
@@ -19,6 +19,11 @@ CLASS zcl_abapgit_utils DEFINITION
         !ev_time    TYPE zif_abapgit_definitions=>ty_commit-time
       RAISING
         zcx_abapgit_exception .
+    CLASS-METHODS translate_postdata
```

Comment:
I'd move it to `zcl_abapgit_html_action_utils` - a more common place for sapevent related stuff.
Not critical though, considering the upcoming #3602

Reply:
Do you have an estimate on the timeline for #3602? 
If it is just around the corner, then I wouldn't bother moving the method, as it then most likely is better suited to be placed wherever the unified event handler is located.
In case the unified event handler is farther off, I'd probably move it so as not to disturb the logical grouping. IN this case we should also think about merging it into master until the refactoring of the event handling, because without the fix it basically blocks all staging and committing (in out case the continued development of our add-on :-)  ).
I've also submitted PR #3625 to at least alleviate the problem in the meantime.

Reply:
I'd certainly merge it. It is a good fix and quick win.
Timeline for #3602: @mbtools, do you have any specific plans for that ?
P.S. moving to `action_utils` - i don't think it is really critical, so up to. Just not to forget it during refactoring :)

Reply:
not sure when I will get to 3602. let's merge this.